# the pixel shader takes every pixel of the grown triangle
--- a/src/airconv/dxbc_converter.cpp
+++ b/src/airconv/dxbc_converter.cpp
@@ -399,7 +399,7 @@
     outside = builder.CreateOr(outside, builder.CreateFCmpOLT(builder.CreateFAdd(value, reach), f(0)));
     inside = builder.CreateAnd(inside, builder.CreateFCmpOGE(builder.CreateFSub(value, reach), f(0)));
   }
-  return {builder.CreateAnd(outside, builder.CreateNot(air.CreateIsNaN(x(0)))), builder.CreateAnd(inside, builder.CreateNot(air.CreateIsNaN(x(0))))};
+  return {builder.getFalse(), builder.CreateAnd(inside, builder.CreateNot(air.CreateIsNaN(x(0))))};
 }
 
 llvm::Error convert_dxbc_pixel_shader(
